### PR TITLE
feat(common): create Kubernetes config files #119

### PR DESCRIPTION
**Pull request by vraybaud**

### CODE REVIEW

**Comment by bobeal** [comment]
@vraybaud what about this PR? reviewable / mergeable?

**Comment by vraybaud** [comment]
> @vraybaud what about this PR? reviewable / mergeable?

This could be reviewable as it is working with minikube
I did this PR only so Haris and his team has direct access to the kubernetes config files.

**Comment by bobeal** [comment]
Can you add a reference in the main README (explaining it is still experimental)?

File: kubernetes/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Tested on a Minikube cluster (for the moment).
```
Comment (bobeal) [inline kubernetes/README.md:1]: at the end, it would deserve a short note saying where (IP / port) you can contact the services

**Pull request merged**